assets/mentions.md

```diff
@@ -0,0 +1,34 @@
+# Mentions of MeiliSearch
```

Comment:
Put the date, order them by ASC date.

Link to the date so links are hidden instead of a list of links.

Reply:
Same for date here 😬